### PR TITLE
Check if the package and UPTANE target got to the backend

### PR DESCRIPTION
**Pull request by OYTIS**

### CODE REVIEW

File: classes/image_types_ostree.bbclass [hunk 1]
```diff
@@ -227,9 +227,20 @@ IMAGE_CMD_garagesign () {
             bberror "Couldn't push to garage repository"
             exit 1
         fi
-    else
-        bbwarn "SOTA_PACKED_CREDENTIALS not set. Please add SOTA_PACKED_CREDENTIALS."
```
Comment (patrickvacek) [inline classes/image_types_ostree.bbclass]: Is there a use case for running this without providing credentials?

File: classes/image_types_ostree.bbclass [hunk 2]
```diff
@@ -227,9 +227,20 @@ IMAGE_CMD_garagesign () {
...
     fi
 }
 
+IMAGE_TYPEDEP_garagecheck = "ostreepush garagesign"
+IMAGE_DEPENDS_garagecheck = "aktualizr-native:do_populate_sysroot"
+IMAGE_CMD_garagecheck () {
+    if [ -n "${SOTA_PACKED_CREDENTIALS}" ]; then
+        # if credentials are issued by a server that doesn't support offline signing, exit silently
+        unzip -p ${SOTA_PACKED_CREDENTIALS} root.json targets.pub targets.sec repo.url 2>&1 >/dev/null || exit 0
```
Comment (cajun-rat) [inline classes/image_types_ostree.bbclass]: Why wont this work when people are using the old (signed on the server) mode?

Comment (cajun-rat) [reply classes/image_types_ostree.bbclass]: sorry, dumb question--the tuf repo is only visible through the device gateway in the 'old world order'

**Comment by cajun-rat** [comment]
Tested with oe-selftest

**Pull request merged**